test-case/check-capture.sh

```diff
@@ -101,5 +102,6 @@ do
     done
 done
 
+pkill -9 arecord
```

Comment:
If the value of $? is 0, then the previous process did terminate normally. It's overkill.